framework/db/Schema.php

```
@@ -560,7 +560,36 @@ public function quoteTableName($name)
+        [$startChar, $endChar] = $this->resolveQuoteCharacter($this->tableQuoteCharacter);
+
+        if (!str_contains($name, $startChar)) {
+            return explode('.', $name);
+        }
+
+        $parts = [];
+        $current = '';
+        $inQuotes = false;
+
+        for ($i = 0, $len = strlen($name); $i < $len; $i++) {
+            $c = $name[$i];
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $c. Configured minimum length is 3. (undefined)

(ShortVariable)